src/main/java/teammates/common/util/Const.java

```diff
@@ -1073,14 +1073,13 @@
         public static final String STUDENT_PROFILE_UNACCESSIBLE_TO_INSTRUCTOR = "Normally, we would show the student’s profile here. "
                 + "However, you do not have access to view this student's profile";
         
+        public static final String UNREGISTERED_STUDENT_MESSAGE = "You may submit feedback for sessions that are currently open "
+                + "and view results without logging in. To access other features you need <a href='%s' class='link'>to login "
+                + "using a Google account</a> (recommended).";
         public static final String UNREGISTERED_STUDENT = "You are submitting feedback as <span class='text-danger text-bold text-large'>%s</span>. " 
-                + "You may submit feedback and view results without logging in. "
-                + "To access other features you need <a href='%s' class='link'>to login using a google account</a> "
-                + "(recommended).";
+                + UNREGISTERED_STUDENT_MESSAGE;
         public static final String UNREGISTERED_STUDENT_RESULTS = "You are viewing feedback results as <span class='text-danger text-bold text-large'>%s</span>. " 
-                + "You may submit feedback and view results without logging in. "
-                + "To access other features you need <a href='%s' class='link'>to login using a google account</a> "
-                + "(recommended).";
+                + UNREGISTERED_STUDENT_MESSAGE;
     }
```

Comment:
While we are at it, can you fix the line length too. Some of these lines are too long.